mix.exs

```diff
@@ -7,24 +7,32 @@ defmodule Mongodb.Mixfile do
     [app: :mongodb,
      version: @version,
      elixir: "~> 1.2",
+     elixirc_paths: elixirc_paths(Mix.env),
      name: "Mongodb",
      deps: deps(),
      docs: docs(),
      description: description(),
      package: package()]
   end
 
+  defp elixirc_paths(:test), do: ["lib", "test/support"]
+  defp elixirc_paths(_),     do: ["lib"]
+
   def application do
-    [applications: [:logger, :connection, :db_connection],
+    [applications: applications(Mix.env),
      mod: {Mongo.App, []},
      env: []]
   end
 
+  def applications(:test), do: [:logger, :connection, :db_connection, :mongoman]
+  def applications(_), do: [:logger, :connection, :db_connection]
+
   defp deps do
     [{:connection,    "~> 1.0"},
      {:db_connection, "~> 1.0"},
      {:ex_doc,        ">= 0.0.0", only: :dev},
-     {:earmark,       ">= 0.0.0", only: :dev}]
+     {:earmark,       ">= 0.0.0", only: :dev},
+     {:mongoman,      "~> 0.3.4", only: [:dev, :test]}]
```

Comment:
Is this needed in `:dev`? If so, it should probably be added to `applications` above.